### DISCUSSION TITLE
Gas discharge

### DISCUSSION THREAD

**Discussion by Vickyhvl** [firemodels/fds#12885] [Q&A]
Hi  everyone, 
I plan to simulate the gas discharge in a room. The gas agent is a mixture of CO2,  Argon and  Nitrogen. The gas agent is released from a cylinder into a room. The mass flow rate, nozzle pressure, cyclinder pressure are given. 

I defined the gas agent as a lumped species
&SPEC ID= 'CARBON DIOXIDE,  LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='ARGON',                  LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='NITROGEN',             LUMPED_COMPONENT_ONLY=.TRUE./
&SPEC ID='IG541', 
      SPEC_ID(1)='NITROGEN',                 VOLUME_FRACTION(1)=0.52,
      SPEC_ID(2)='ARGON',                      VOLUME_FRACTION(2)=0.40,
      SPEC_ID(3)='CARBON DIOXIDE',     VOLUME_FRACTION(3)=0.08/

I want to use a nozzle to release particles whose SPEC_ID='IG541' . According to FDS user guide, particles cannot evaporate to a lumped species. I just wonder if it is possible to use nozzle to release a lumped species? If not, do you have any alternative solutions?   

Thanks in advance

Vicky

**Comment by rmcdermo** [comment]
Is there any reason why you must use particles?  You can just inject the gas as a gas through a VENT.  Create a VENT at the location where you want the nozzle, the give it a SURF_ID with the MASS_FLUX of SPEC_ID='IG541'.  You can also assign this SURF_ID to a STATIC particle if the direction of the release is not important.

**Comment by Vickyhvl** [comment]
Hi,
Thank you for your quick reply. 
If particles can not release lumped species, I will try to use vent to release gas. 
As the discharge rate is very large. Max mass flow rate =0.92 kg/s., which is a strong jet flow.  Maybe I have to define a big vent and use very fine mesh in the vicinity of the vent. 
Very appreciate your suggesion. 
Vicky

**Comment by mcgratta** [comment]
Are you interested in the details of the flow in the immediate vicinity of the nozzle? If so, you might need a fine mesh. If not, just inject the gas from a 1 cell wide vent. Try some simple test cases before you develop the full input file.

**Comment by drjfloyd** [comment]
If this is a release from a high pressure cylinder, then the flow at the release point is likely choked flow (Mach > 0.3). If details in the compressible flow region are important, then FDS may not be the code you want to use. Otherwise, depending on exactly what results you need from the FDS simulation, you may need to think about an approach that preserves both momentum and species flux.

**Comment by Vickyhvl** [comment]
Hi, 
The compressible flow region is not what I concern.  I will use a cell wide vent first to see the performance of the simulation.
Many thank to you both
Vicky
